src/main/java/com/powsybl/openloadflow/network/LfBus.java

```diff
@@ -40,6 +40,12 @@
 
     double getLoadTargetP();
 
+    void setLoadTargetP(double loadTargetP);
+
+    boolean isParticipating();
```

Comment:
I would not add isParticipating and getParticipationFactor to bus. Is is quite confusing with the one at generator level and in case of load distributed slack you only need to have  getLoadTargetP + setLoadTargetP + getLoadCount (to add) available to implement the algo.

Reply:
Done.